RELEASE/scripts/autoscend/paths/kingdom_of_exploathing.ash

```diff
@@ -91,27 +96,86 @@ boolean LX_koeInvaderHandler()
 	int turns = ceil(0.95 / damagePerRound);
 
 	int damageCap = 100 * my_daycount();
-
+	
+	// It's easiest to kill the invader in two hits with the June cleaver and Lunging Thrust Smack
+	if(have_skill($skill[Lunging Thrust-Smack]) && auto_is_valid($skill[Lunging Thrust-Smack]) &&
+	   auto_canUseJuneCleaver())
+	{
+		// To kill in 3 rounds, need 19 of each element, or 10 plus bend hell. Check we have it.
+		boolean have_19_each = true;
+		boolean have_10_each = true;
+		foreach el in "Cold;Hot;Sleaze;Spooky;Stench".split_string(";")
+		{
+			int damage_bonus = to_int(get_property("_juneCleaver"+el));
+			if (damage_bonus < 19) { have_19_each = false; }
+			if (damage_bonus < 10) { have_10_each = false; }
+		}
+		// Cast bend hell if needed
+		if (have_10_each)
+		{
+			if (have_skill($skill[Bend Hell]) && auto_is_valid($skill[Bend Hell]) &&
+			   !have_19_each)
+			{
+				buffMaintain($effect[Bendin\' Hell]);
+			}
+			if (have_effect($effect[Bendin\' Hell])>0)
+			{
+				have_19_each = true;
+			}
+		}
+		
+		// Actually go in for the kill
+		if (have_19_each)
+		{
+			acquireMP(24, 0);
+			auto_log_info("Attacking the Invader, using June Cleaver and LTS.", "blue");
+			addToMaximize("200 all res, +equip june cleaver");
+			boolean ret = autoAdv(1, $location[The Invader]);
+			if(have_effect($effect[Beaten Up]) > 0)
```

Comment:
I know Soolar had originally added this code, but since we are updating support anyway. Using the spaceInvaderDefeated preference might be better to figure out if we beat the Invader or not. I beat the Invader, but got Beaten Up by passive damage after the fight ended which made me get Beaten Up, making autoscend think that I died to the invader, which I technically did, but I beat it too.

Reply:
Actually, I see you added spaceInvaderDefeated higher up too so probably just need to add an additional clause to this conditional, something like

&& !get_property("spaceInvaderDefeated").to_boolean()